ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -1605,4 +1604,20 @@ FCU_DFSS_DFVSC_FDPM_CHWDC_SRC_DFRMM:
   - FDPM
   - CHWDC
   - SRC
-  - DFRMM 
\ No newline at end of file
+  - DFRMM
+
+FCU_RHDHS_DTM_DFSS_STM_DTC_SHM_SFVSC_ALM_SS_SRC:
```

Comment:
Requires coil control fields as well. Expecting CHWDC or DXDC, or similar type indicating cooling control (unless heating-only, in which case would expect analogous types for heating). Please fix.

Reply:
noted sir. I have seen DXSC for supply and then I have return air pointslist